wiki/Article_styling_criteria/Formatting/de.md

```diff
@@ -0,0 +1,782 @@
+
+```yaml
+outdated: true
+```
+
+Sofern dieses Tag zu einem Artikel hinzugefügt wird, sollten [Kommentare](#kommentare) eingefügt werden, die erklären, was bei diesem Artikel überarbeitet werden muss.
+
+### Artikel mit Tags versehen
+
+Tags helfen der Suchmaschine der Webseite relevantere Artikel anzuzeigen. Tags sollten in derselben Sprache, wie der Artikel geschrieben sein und zusätzlich noch die originalen Tags. Tags sollten nach Möglichkeit immer Kleinschreibung verwenden.
+
+Ein Artikel über "Beatmap-Diskussion" könnte beispielweise diese Tags enthalten:
+
+```yaml
+tags:
+  - beatmap discussions
+  - modding V2
+  - MV2
+  - beatmap-diskussionen
+```
+
+### Übersetzungen ohne Überprüfungen
+
+*Tipp: Wiki-Verwalter werden diese Markierung vor dem Mergen setzen, falls notwendig.*
+
+Manchmal werden Artikel ohne eine Überprüfung eines Muttersprachlers dem Wiki hinzugefügt. In diesem Fall muss die Markierung `no_native_review` gesetzt werden, damit zukünftige Übersetzer wissen, dass der Artikel gegebenenfalls nochmals überprüft werden muss. Dieses Tag muss folgendermaßen geschrieben werden:
+
+```yaml
+no_native_review: true
+```
+
+## Artikelbenennung
+
+*Siehe auch: [Ordnernamen](#ordnernamen) und [Titel](#titel)*
+
+Artikeltitel sollten im Singular stehen und "Sentence case" verwenden. Siehe [Wikipedias Artikel über die Namensgebung](https://de.wikipedia.org/wiki/Wikipedia:Archiv/Richtlinien_f%C3%BCr_die_Namensgebung:Singular-Plural-Debatte) für mehr Details.
+
+Artikeltitel sollten dem Ordnernamen entsprechen, in welchem sie sich befinden (Unterstriche (`_`) dürfen gerne an angemessenen Stellen mit Leerzeichen ersetzt werden). Wenn der Ordnername sich ändert, dann sollte der Titel sich ebenfalls ändern und umgekehrt.
+
+---
+
+Wettbewerbsartikel sind eine Ausnahme. Der Ordner muss Abkürzungen, Akronyme oder Initialien verwenden. Der Titel des Artikels muss den vollen Namen des Wettbewerbs enthalten.
+
+## Ordner- und Dateistrukturen
+
+### Ordnernamen
+
+*Siehe auch: [Artikelbenennung](#artikelbenennung)*
+
+Ordnernamen müssen in Englisch sein und "Sentence case" verwenden.
+
+Ordnernamen dürfen nur diese Zeichen enthalten:
+
+- Groß- und Kleinbuchstaben
+- Zahlen
+- Unterstriche (`_`)
+- Bindestriche (`-`)
+- Ausrufezeichen (`!`)
+
+### Dateinamen für Artikel
+
+Der Dateiname für einen Artikel kann in der Spalte `Dateiname` der Tabelle in [Locales](#locales) gefunden werden. Der übersetzte Artikel muss in denselben Ordner wie der englische Artikel abgelegt werden.
+
+### Indexartikel
+
+Ein Indexartikel muss erstellt werden, wenn der Ordner nur dazu gedacht ist, andere Artikel zu beinhalten. Indexartikel müssen eine Liste an den enthaltenen Artikeln haben. Sie können auch weitere Informationen enthalten, z. B. einen Hauptabsatz oder eine Beschreibung der verlinkten Artikel.
```

Comment:
```suggestion
Ein Indexartikel muss erstellt werden, wenn der Ordner nur dazu gedacht ist, andere Artikel zu beinhalten. Indexartikel müssen eine Liste an den enthaltenen Artikeln haben. Sie können auch weitere Informationen enthalten, z. B. einen Hauptabsatz oder eine Beschreibung der verlinkten Artikel.
```
Amazing, this is a [non-breaking space](https://en.wikipedia.org/wiki/Non-breaking_space), which prevents an automatic line break. Yeah, also LanguageTool's idea, not mine xD

Reply:
The world of formatting and special characters is wonderful.